string_permutation_problem/permute.py

```diff
@@ -0,0 +1,24 @@
+
+def permute_my_string(my_str):
+    output_string = []
```

Comment:
This is not output **string**. It is a list of strings. Please name it accordingly.